packages/asc-ui/src/components/Content/FlexContentStyle.ts

```diff
@@ -0,0 +1,39 @@
+import styled from '@datapunt/asc-core'
+import ContentStyle, { Props as ContentStyleProps } from './ContentStyle'
+import { customCss } from '../../utils'
+
+export interface Props extends ContentStyleProps {
+  wrapReverse?: string
+  noWrap?: string
+  justifyContent?: string
+  justifyCenter?: string
+  alignItems?: string
+  alignCenter?: string
+  alignContent?: string
+  column?: string
+}
+
+export default styled(ContentStyle)<Props>`
```

Comment:
Feels like bloating the codebase to, would make sense if we have default margins/paddings defined in the theme that we can pass

Reply:
refactored